### PR TITLE
OEL-1133: Dependency updated to whitelabel contact forms submodule.

### PR DESCRIPTION
**Pull request by Maxfire**
### Description

Info.yml updated to enable a dependency with the oe_whitelabel_contact_forms submodule.

### CODE REVIEW

**Comment by Maxfire** [comment]
Failing because requires https://github.com/openeuropa/oe_whitelabel/pull/73#pullrequestreview-886905313 to be merged 1st so that the module oe_whitelabel_contact_forms exists.

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
+        "php": ">=7.4",
```
Comment (drishu) [inline composer.json:9]: for oe_contact_forms can we use the branch "dev-OEL-..", after it gets merged we can use a dev release till the official one
and for oe_whitelabel, again you can use the branch in the patch https://github.com/openeuropa/oe_whitelabel/compare/0.1.202202080830...OEL-1133.patch and of course you done use latest 1.x as label but something that makes sense

**Review by drishu — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Maxfire) [reply composer.json:9]: JFYI I did add master on oe_contact_forms cauyse OEL-1134 was merged.

File: composer.json [hunk 2]
```diff
@@ -113,7 +113,7 @@
                 "latest 1.x": "https://github.com/openeuropa/oe_bootstrap_theme/compare/0.1.202202072010...1.x.patch"
             },
             "openeuropa/oe_whitelabel": {
-                "latest 1.x": "https://github.com/openeuropa/oe_whitelabel/compare/0.1.202202080830...1.x.patch"
+                "OEL-1133-theming branch": "https://github.com/openeuropa/oe_whitelabel/compare/0.1.202202080830...OEL-1133-theming.diff"
```
Comment (drishu) [inline composer.json]:
```suggestion
                "contact forms theming": "https://github.com/openeuropa/oe_whitelabel/compare/0.1.202202080830...OEL-1133-theming.patch"
```

**Review by drishu — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (Maxfire) [reply composer.json]: We have to use .diff temporally to avoid error on drone.

**Review by drishu — DISMISSED** [review]

**Review by abel-santos-corral — DISMISSED** [review]

**Review by abel-santos-corral — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Pull request merged**